app/services/invite_schools.rb

```diff
@@ -230,6 +230,25 @@ def send_year2020_invite_email
     end
   end
 
+  def invite_cip_only_schools
+    # We want to exclude sending invites to welsh schools (code 30), which is the only other cip only
+    # type code in GiasTypes::CIP_ONLY_TYPE_CODES
+    School.currently_open.where(school_type_code: [10, 11, 37]).where(section_41_approved: false).find_each do |school|
```

Comment:
Can we put that in `GiasHelpers`? 
At least the codes, but perhaps also the general scope, which means `cip_only_except_welsh`

Reply:
Ok, the reason I didn't do that is because its going into an email that we send once (we should be able to delete this code after we send the invites. But I can do it if you prefer

Reply:
Yeah, that makes sense, but I really don't want to see school type codes anywhere outside of closed and isolated modules, cause they are 'magic numbers'.

Feel free to add `TODO` comments around the code that tells us to delete it when we no longer need it :)